Formula/llvm.rb

```diff
@@ -37,6 +37,12 @@ class Llvm < Formula
       url "https://raw.githubusercontent.com/Homebrew/formula-patches/6166a68c/llvm/openmp_arm.patch"
       sha256 "70fe3836b423e593688cd1cc7a3d76ee6406e64b9909f1a2f780c6f018f89b1e"
     end
+
+    # Regression in LLDB by D89156 on MacOS, fixed by D95683.
```

Comment:
```suggestion
    # Regression in LLDB by D89156 on MacOS, fixed by D95683.
    # https://reviews.llvm.org/D95683
```
Please still include the link, thanks.

Reply:
OK